pkg/skaffold/cluster/minikube.go

```diff
@@ -89,22 +104,60 @@ func (clientImpl) MinikubeExec(arg ...string) (*exec.Cmd, error) {
 }
 
 func minikubeExec(arg ...string) (*exec.Cmd, error) {
-	b, err := minikubeBinaryFunc()
+	b, v, err := minikubeBinaryFunc()
 	if err != nil {
 		return nil, fmt.Errorf("getting minikube executable: %w", err)
 	}
+
+	isSupported, _ := supportsUserFlag(v)
+	if isSupported {
+		arg = append(arg, "--user=skaffold")
```

Comment:
does `arg = append(arg, "--user", "skaffold")` not work? I think it's preferred to have each arg be a single slice element.

Reply:
That seem to add a space in-between `--user` & `skaffold`.